### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by rchavan10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
-# hra-ui
-Human Reference Atlas UIs Monorepo
+# Human Reference Atlas (HRA) and Functional Tissue Unit Explorer (FTU) - User Interfaces
+
+[![Tests]()](https://github.com/hubmapconsortium/hra-ui/actions/workflows/ci.yml)
```
Comment (bherr2) [inline README.md:3]: Please fix this so it shows the badges

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+[![GitHub last commit]()](https://github.com/hubmapconsortium/hra-ui/commits/develop)
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg)](LICENSE)
+[![Quality Gate Status]()](https://sonarcloud.io/project/overview?id=hubmapconsortium_hra-ui)
+[![SonarCloud Coverage]()](https://sonarcloud.io/component_measures?metric=Coverage&id=hubmapconsortium_hra-ui)
+[![Documentation Status]()]()
+
+## Website Links
+
+- FTU User Interface (FTU)
+  - Production: <>
```
Comment (bherr2) [inline README.md:13]: Production (alt) is production, so remove (alt) from below and remove this empty line

File: README.md (hunk 1 above)
Comment (rchavan10) [reply README.md:3]: Where can I get the links to the badges for hra-ui?

Comment (bherr2) [reply README.md:3]: Follow the model from the thing you _literally_ copy and pasted from https://raw.githubusercontent.com/hubmapconsortium/ccf-ui/main/README.md

**Comment by bherr2** [comment]
You have combined two PRs here

**Comment by rchavan10** [comment]
I will create a new PR for the screen notice thing.

Best,
Raj

**Review by bherr2 — APPROVED** [review]

**Pull request merged**